### ISSUE TITLE
Create query parameters from class/struct

### ISSUE THREAD

**Issue by 4brunu** [SRombauts/SQLiteCpp#129]
Would it be possible to create the opposite of https://github.com/SRombauts/SQLiteCpp/pull/114, by creating an array of parameters, based on a class/struct?
Maybe with templates also?
I'm asking, because I would really appreciate it, and could try to implement it, but I'm a bit lost, and trying to find some guidance, or know if it's possible at all.
@SRombauts @dend456 
Thanks

**Comment by SRombauts** [comment]
Yes it's a very good question. Solving this is similar to serialization; we would like to be able to write the content of an arbitrary structure to another format. This is not trivial in C++ without reflexion features...

Heavy templates like those in Boost could do the trick (but don't ask me to write these ;) or a serialization framework could help (Qt?)

**Comment by SRombauts** [comment]
Closing this now due to age and inactivity; this broad struct-to-query mapping feature can be proposed again if still useful.
